enterprise/backend/src/metabase_enterprise/serialization/v2/extract.clj

```diff
@@ -62,17 +62,22 @@
     (eduction (map #(serdes/extract-all % extract-opts)) cat (model-set opts))))
 
 ;; TODO Properly support "continue" - it should be contagious. Eg. a Dashboard with an illegal Card gets excluded too.
-(defn- descendants-closure [targets]
+(defn- chase-for [targets func]
   (loop [to-chase (set targets)
          chased   #{}]
     (let [[m i :as item] (first to-chase)
-          desc           (serdes/descendants m i)
+          res            (func m i)
```

Comment:
`res` could be more specific. This is graph traversal, so we are getting adjacent nodes here. I would call this `adjacents` or something